### ISSUE TITLE
Feature Request: Schedule Task on Specific Date/Time

### ISSUE THREAD

**Issue by bradhawkins85** [wh1te909/tacticalrmm#109]
Can you please add the ability to schedule tasks to run One Time on a specific date and time?
This will be useful to schedule disabling user accounts etc.
Also, allow setting the option on the scheduled task to run task immediately if scheduled start is missed.
Assuming these could also be scheduled as part of automation manager tasks it would allow for an "On First Check In" style tasks to perform basic initial setup tasks.

**Comment by sadnub** [comment]
@bradhawkins85 I added a PR that allows for Run Once tasks for policies and agents. You can also optionally have the task auto delete itself after the scheduled run date. Once it is merged test it out

**Comment by bradhawkins85** [comment]
Thank you @sadnub 
Did you get a chance to add the option on the task to run task immediately if scheduled start is missed?

**Comment by wh1te909** [comment]
@bradhawkins85 I created separate ticket for the schedule missed since when I merge this PR it will auto close this ticket
